main/src/main/java/cgeo/geocaching/activity/AbstractBottomNavigationActivity.java

```diff
@@ -249,7 +250,10 @@ public void updateSelectedBottomNavItemId() {
     }
 
     private void setCustomBNitem() {
-        final MenuItem menu = ((BottomNavigationView) binding.activityBottomNavigation).getMenu().findItem(MENU_NEARBY);
+        final MenuItem menu = (binding.activityBottomNavigation instanceof  NavigationRailView
```

Comment:
Probably you can just use [NavigationBarView](https://developer.android.com/reference/com/google/android/material/navigation/NavigationBarView) which is the superclass of both

Reply:
A typical case of bad class naming. And of "I should have RTFM" ;-) Thanks for the pointer. Fixed with #14469.

Reply:
> Maybe we should also rename our layout resource id from `activityBottomNavigation` to something which sounds a bit more like NavigationBarView?

Ok, but let's do this in a separate PR. 

Proposal would be:
- name the layout files `activity_navigation.xml` instead of `activity_bottom_navigation.xml`,
- and name the navigation component part within those files `navigationBarView` (or maybe with underlines) instead of `activity_bottom_navigation`

Reply:
See #14484 for renaming of files / view id